script/e2e-test.sh

```diff
@@ -339,6 +368,10 @@ kubectl create serviceaccount kubeapps-view -n kubeapps
 kubectl create role view-secrets --verb=get,list,watch --resource=secrets
 kubectl create rolebinding kubeapps-view-secret --role view-secrets --serviceaccount kubeapps:kubeapps-view
 kubectl create clusterrolebinding kubeapps-view --clusterrole=view --serviceaccount kubeapps:kubeapps-view
+## Create view user (oidc)
```

Comment:
and the admin user?

Reply:
It is already created with cluster-admin role with the default RBAC (`kubeapps-operator@example.com`). 
This step is just modifying the `kubeapps-user@example.com`, but we can perform this step in the same place as the operator user for cohesion if you may prefer.

Reply:
I see, yes, better to have them in the same place.